src/pages/AI/components/Chat/docgen/uploadDocs.jsx

```
@@ -0,0 +1,842 @@
+                        </button>
+                        <button className="primary-button" onClick={handleNext}>
+                            {step === 3 ? 'Submit' : 'Next'}
+                        </button>
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+};
+
+export default AddUser;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)